embc-app/ViewModels/PrintReferral.cs

```diff
@@ -122,6 +122,14 @@ private string ConvertCarriageReturnToHtml(string value)
         {
             return value?.Replace("\n", "<br />")?.Replace("\r", "<br />");
         }
+
+        private string GetLocalTime(DateTime dateTime)
+        {
+            var pst = TimeZoneInfo.FindSystemTimeZoneById("Pacific Standard Time");
```

Comment:
Will this handle PDT conversion as well?

Reply:
Yes, it will.

Reply:
Each timezone either supports or doesn't support PDT conversion.

Reply:
PST does